etc/databases/world/updates/updates.sql

```diff
@@ -3292,5 +3292,39 @@ begin not atomic
     end if;
 
 
+    if (select count(*) from `applied_updates` where id='010820231') = 0 then
+        --01/08/2023
+        --mostly devwr4 teaching klorine28 what to do (thanks)
+        >>> static_flags = 268959760  <- Current Flags
+        >>> static_flags |= 262144    <- Add AQUATIC
+        >>> static_flags &= ~524288   <- Remove AMPHIBIOUS
+        
+        static_flags:
+        268697616
```

Comment:
3297-3303 lines won't even run. Do you have a testing environment set up so you can try everything before submitting it?

Reply:
yeah i use vs code